src/common.ts

```diff
@@ -147,6 +147,7 @@ export namespace CommonNames {
   export const indexof = "indexof";
   export const valueof = "valueof";
   export const returnof = "returnof";
+  export const notnullable = "NonNullable";
```

Comment:
Wdyt of naming the built-in type just "nonnull" to align with hat is already there, and make `NonNullable` an alias of it in stdlib, similar to what is done for `type ReturnType<T> = returnof<T>`?

Reply:
Done!